.eslintrc

```diff
@@ -33,6 +33,11 @@
     "react"
   ],
   "rules": {
+    "import/no-absolute-path": 0,
```

Comment:
Are these changes really relevant for purposes of this PR?

Reply:
Yes , they are necessary.